src/app/timeline/TimeLine.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
-import { Survey, SurveyTimeLine } from '../components/survey/Survey'
+import { Survey, SurveyTimeLine } from '../../components/survey/Survey'
 
 interface TimeLine {
   surveyList: Survey[];
 }
 
-const mockSurveys = require("../mockSurveyData.json");
+const mockSurveys = require("../../mockSurveyData.json");
```

Comment:
please revert this change this will lead to conflict this branches only purpose is to create routing mechanism.